corehq/doctypemigrations/management/commands/run_doctype_migration.py

```diff
@@ -79,3 +113,17 @@ def handle_continuous(self, migrator):
                 status_update.changes_read, status_update.last_seq))
             if status_update.caught_up:
                 self.stdout.write('All caught up!\n')
+
+    def handle_cleanup(self, migrator):
+        migrator.phase_3_clean_up()
+
+    def handle_stats(self, migrator):
+        [(source_db, source_counts),
+         (target_db, target_counts)] = migrator.get_doc_counts()
+        self.stdout.write('Source DB: {}\n'.format(source_db.uri))
+        self.stdout.write('Target DB: {}\n'.format(target_db.uri))
+        self.stdout.write('{:^30}\tSource\tTarget\n'.format('doc_type'))
```

Comment:
It seems to be the convention is all https://docs.djangoproject.com/en/1.7/howto/custom-management-commands/#management-commands-output. Although apparently the newlines aren't necessary as I just learned from reading that.